WooCommerce/Classes/POS/Presentation/Barcode Scanner Setup/PointOfSaleBarcodeScannerSetupStepViews.swift

```diff
@@ -148,23 +177,29 @@ struct PointOfSaleBarcodeScannerSetupCompleteView: View {
     @ViewBuilder private var successIcon: some View {
         ZStack {
             Circle()
-                .frame(width: 104, height: 104)
+                .frame(width: 88, height: 88)
                 .foregroundColor(.posSuccess)
             Image(PointOfSaleAssets.successCheck.imageName)
                 .renderingMode(.template)
                 .resizable()
-                .frame(width: 48, height: 48)
+                .frame(width: 36, height: 36)
                 .foregroundColor(.posOnSuccess)
         }
     }
 }
 
 private extension PointOfSaleBarcodeScannerSetupCompleteView {
     enum Localization {
-        //TODO: WOOMOB-792
-        static let title = "Scanner set up!"
-        static let instruction = "You are ready to start scanning products. \n" +
-        "Read more about barcode and QR code scanner support."
+        static let title = NSLocalizedString(
+            "pos.barcodeScannerSetup.complete.title",
+            value: "Scanner set up!",
+            comment: "Title shown when scanner setup is successfully completed"
+        )
+        static let instruction = NSLocalizedString(
+            "pos.barcodeScannerSetup.complete.instruction",
+            value: "You are ready to start scanning products. \nRead more about barcode and QR code scanner support.",
```

Comment:
super nit: unless the space at the end of the first line is intentional.
```suggestion
            value: "You are ready to start scanning products.\nRead more about barcode and QR code scanner support.",
```

Reply:
It was intentional 👍